user-profiles.md

```diff
@@ -0,0 +1,49 @@
+
+## Preface
+
+* What's described here are user profiles as opposed to [personas](https://en.wikipedia.org/wiki/Persona#In_user_experience_design). Personas are example actors rather than general categories.
+* Kubernetes Cluster Operators are out of scope for this document. A cluster operator is one who manages the operation of a Kubernetes cluster that applications can run in.
+* Application Operators are in scope for this document at a lower priority than Pipeline Operators. An application operator is one who manages the operation of the application in a Kubernetes cluster.
+
+## Profiles
+
+Profiles describe a type of role a user may perform. A real person may perform more than one role and have more than one profile apply to them. How this mapping works between profiles and real people can vary between companies and other organizations. To handle this variation we focus on the user profiles rather than how they may map to people in these different organizations.
+
+### 1. Pipeline Operator
+
+Pipeline operators define, schedule, monitor, and otherwise operate pipelines across one or more Kubernetes clusters. For example, the operation of a set of Kaniko build pipelines and Argo CD pipelines. This is not to be confused with the role of a Kubernetes cluster operator. This is also not to be confused with the application's operator role, who would oversee a product in production.
+
+### 2. Pipeline Distributor
+
+Distributors are people who package an application and application pipelines for someone else to operate. Examples of this would be those who maintain [Catalog Pipelines](https://github.com/tektoncd/catalog) such as for Kaniko and ArgoCD.
+
+### 3. Application Operator and Application Distributor
+
+Application operators take an application and operate it within a Kubernetes cluster. For example, the operation of WordPress and MySQL. This is not to be confused with the role of a Kubernetes cluster operator (see earlier in preface for more detail). Application distributors are people who package an application for someone else to operate. Examples of this would be those who maintain community Helm Charts for WordPress and Datadog. These are the same profiles as seen in Helm's profile document https://github.com/helm/community/blob/master/user-profiles.md
+
+### 4. Application Developer
+
+An application developer writes the software for an application. They are typically not concerned with where the application is running as applications can often be run more than one environment (e.g., many applications can be run in a virtual machine or a container). Examples of this include the developers of WordPress and MySQL. These developers are also typically not concerned with what pipelines run to produce or ship an application, as long as the pipeline meets organizational needs (e.g., for signoff, security review, or other business workflows).
```

Comment:
> These developers are also typically not concerned with what pipelines run to produce or ship an application, as long as the pipeline meets organizational needs (e.g., for signoff, security review, or other business workflows).

I tend to disagree with that, as they should care about their CI/CD.

Reply:
:thinking: Which parts of their Tekton pipelines will Application Developers like to understand and see? For example, perhaps the developer "should" care about K8S pods, though "should not be required" to care how PVC or pods work in order to depend on them. Do you mean the AppDev shall care about the steps, debug information, health of build, and hooks from CI/CD into their "get it to production" pipeline/workflow? Might discuss in Slack as well

Reply:
Added a comment about a single person filling multiple roles, or matching on multiple profiles to the Preface

"* What's described here are user profiles as opposed to [personas](https://en.wikipedia.org/wiki/Persona#In_user_experience_design). Personas are example actors rather than general categories. **A single person can potentially match with multiple user profiles at the same time.**"

Reply:
Reworded section on what an Application Developer cares about to not include what the AppDev does not care about, leaving it to omission of what the Dev is not focused on doing (though could potentially care about)

Reply:
"An application developer is focused on releasing product updates with high quality as quickly as possible. Examples of this include the developers of WordPress and MySQL."

Priorities within TektonCD that make AppDev's jobs easier should be included. Perhaps release tagging mechanisms, "skip a bunch of stages and go straight to production", or "minor CSS change without extensive testing ships quickly to staging" features as example. These may not benefit the TektonCD workflow community, though when a real developer comes in with a few real problems and asks if Tekton can assist or add a few flags (or extend our plugin model/update a controller) to support something, it can be considered at a lower priority than Operators' concerns (per the outlined order of concerns currently)